src/crud.js

```diff
@@ -0,0 +1,34 @@
+export const add = (todos, item) => {
+  todos.push({
+    description: item,
+    index: todos.length + 1,
+    completed: false,
+  });
+  window.localStorage.setItem('todos', JSON.stringify(todos));
```

Comment:
Optional: Do not use window.alert() or window.confirm(). It's ugly, impossible to style, it stops code execution and displays differently on different browsers. Use custom modal instead.